content/deployment/reference/server-specifications.md

```diff
@@ -5,27 +5,45 @@ weight: 2
 
 # Server specifications
 
-The minimum recommended server specifications are:
+The minimum server specifications are:
 
-- 2GB RAM
-- 1CPU @ 2GHz
-- 10GB storage
+- 2 GB RAM
+- 1 CPU @ 2GHz
+- 2 GB storage
 
-Approximately 750MB of storage is required for the engine itself, with the remaining space allocated for data generated by the engine.
+Approximately 750 MB of storage is required for the engine itself, with the remaining space allocated for data generated by the engine. Storage capacity should be calculated based on the scope and expected lifetime of the collection.
 
-## Storage growth
+## Storage
 
-Storage requirements typically grow at a rate of less than 0.5MB per tracked terms per month on average. This growth rate varies depending on the type of service being tracked:
+Recommended storage: 10 GB, with resizable partitioning to enable growth over time.
 
-- Very large online platforms with large legal teams and heavy layouts: ~1MB per terms per month
-- Standard collections: ~0.1-0.3MB per terms per month
+Storage usage typically grows at a rate of 0.5 MB per tracked terms per month on average for latin scripts. This growth rate varies depending on the type of service being tracked:
+
+- Services with large legal teams and heavy website layouts: ~1 MB per terms per month
+- Smaller services: ~0.1-0.3 MB per terms per month
+
+### Examples
+
+- Tracking 5 very large social media platforms on their 5 most common terms types (such as Terms of Service, Privacy Policy, Trackers Policy, Developer Agreement, Community Guidelines) would require approximately 300 MB of additional storage per year.
+- Tracking a sample of 200 platforms on an average of 4 common terms types would require approximately 5 GB of additional storage per year.
 
 ## Operating system
 
-Supported operating systems:
-- Ubuntu 20.04 or higher
-- Debian 11 or higher
+The Open Terms Archive engine is [continuously tested](https://github.com/OpenTermsArchive/engine/blob/main/.github/workflows/test.yml#L12-L15) and guaranteed to work on the following operating systems:
+
+- Ubuntu 22.04
+- macOS 14
+- Windows Server 2022
+
+The following operating systems should be compatible, but are not guaranteed:
+
+- Debian 11 or higher
```

Comment:
I know it is not in out tests but all the collections we administrate are deployed on Debian 11, so I think we can guarantee that it works on that operating system